v2/emailpassword/email-delivery/smtp/configure-smtp.mdx

```diff
@@ -132,4 +135,8 @@ init(
 </TabItem>
 </BackendSDKTabs>
 
+:::important
+The `username` is an optional field. In case it's not provided, `from.email` is used as the default auth username.
+:::
```

Comment:
I think it would be better to add this as a comment in the code snippet above (suggested it as well), since people may miss out on reading this.